tunnel-server/src/tunnel-store.ts

```diff
@@ -54,3 +55,19 @@ export const inMemoryActiveTunnelStore = ({ log }: { log: Logger }): ActiveTunne
     },
   }
 }
+
+const MAX_DNS_LABEL_LENGTH = 63
+
+export const activeTunnelStoreKey = (clientId: string, remotePath: string) => {
+  // return value needs to be DNS safe:
+  // - max DNS label name length is 63 octets (== 63 ASCII chars)
+  // - case insensitive
+  const tunnelPath = remotePath.substring(1)
```

Comment:
this removes the leading `/`? if so, it's duplicated below by `x.replace(/^\//, '')`
IMO the latter is better since it clarifies the purpose.

Reply:
fixed